Zilon.Core/Zilon.Core/Tactics/GameLoop.TaskState.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using Zilon.Core.Tactics.Behaviour;
+
+namespace Zilon.Core.Tactics
+{
+    public sealed partial class GameLoop
+    {
+        private class TaskState
+        {
+            private readonly int _valueToExecute;
+
+            public TaskState(IActor actor, IActorTask task, IActorTaskSource taskSource)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private constructor 'TaskState'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5544638)